distributed/http/scheduler/api.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+import json
+
+from distributed.http.utils import RequestHandler
+
+
+class APIHandler(RequestHandler):
+    def get(self):
+        self.write("API V1")
+        self.set_header("Content-Type", "text/plain")
+
+
+class RetireWorkersHandler(RequestHandler):
+    async def post(self):
+        self.set_header("Content-Type", "text/json")
+        scheduler = self.server
+        try:
+            params = json.loads(self.request.body)
+            workers = params["workers"]
+            workers_info = await scheduler.retire_workers(workers)
```

Comment:
We probably want to support retiring a number of workers too rather than just by name.